drivers/wifi/nrf700x/Kconfig.nrf700x

```diff
@@ -102,4 +102,8 @@ config RX_MAX_DATA_SIZE
 	int "Maximum size of RX data"
 	default 1600
 
+config NRF700X_SCAN_LIMIT
+	int "Maximum number of scan results returned to application. Use negative values for unlimited scan results."
```

Comment:
So this is not handling any ordering but we randomly drop any additional results that may be found, right? I guess  the results are in the order they are found by the device so from user perspective it's random.

Reply:
The ordering part has been addressed by https://github.com/nrfconnect/sdk-nrf/pull/10184.

Reply:
Great, so ordering is handled in the driver.

Reply:
@sachinthegreen and just to be on the same page 1) the ordering is always done by RSSI regardless of other parameters like band or security level, right? 2) So the higher RSSI always comes before the lower one as a scan result? 3) And I presume no additional tweak are needed on the net_mgmt nor application layer (this should work out of the box).

Reply:
@markaj-nordic : Yes to all the questions.